src/csharp/Extensions/Microsoft.Spark.Extensions.FileSystem/FileSystem.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.Extensions.FileSystem
+{
+    /// <summary>
+    /// An abstract base class for a fairly generic filesystem. It may be implemented as a distributed filesystem, or
+    /// as a "local" one that reflects the locally-connected disk. The local version exists for small Hadoop instances
+    /// and for testing.
+    /// 
+    /// All user code that may potentially use the Hadoop Distributed File System should be written to use an FileSystem
+    /// object. The Hadoop DFS is a multi-machine system that appears as a single disk.It's useful because of its fault
+    /// tolerance and potentially very large capacity.
+    /// </summary>
```

Comment:
Please reformat to keep each line within the 110 character limit

Reply:
Oops, I thought we increased it to 120 for some reason.  Fixed.